### DISCUSSION TITLE
Strength of links from PCMCI

### DISCUSSION THREAD

**Discussion by Agatheee** [jakobrunge/tigramite#232] [Q&A]
The PCMCI is giving the (parent) links but not the coefficients or the strengths. Even there is not a variable within the function that can get the parent links like in the ```pcmci_parcorr.run_pc_stable```.
Is there any way to get those? I saw in tutorial that one way to look into the strengths is to do something as:

```
med = LinearMediation(dataframe=dataframe)
med.fit_model(all_parents=true_parents, tau_max=4)
```

and then look into the ```med.psi```. But I am eager to use the PCMCI model, since there might be couple of issues rising from my dataset. So essentially, is there any way to get the weighted adjacency matrix from the PCMCI itself? If not, what about the binary matrix?

An additional question I have is if this obtained adjacency matrix (say ```med.psi```) is in any way comparable to the VAR(p) model coefficients, that we input at the beginning.

Thank you for the guidance!

**Comment by jakobrunge** [comment]
``val_matrix`` may be what you want, but its interpretation is not a causal effect. The test statistic values (e.g., partial correlation) may give a qualitative intuition of the strength of a dependency, but for a proper causal effect analysis please refer to the CausalEffects class and tutorial.
